src/script/util/DebugUtil.js

```diff
@@ -31,6 +32,9 @@ import {BackendEvent} from '../event/Backend';
 import {StorageSchemata} from '../storage/StorageSchemata';
 import {EventRepository} from '../event/EventRepository';
 
+import {h, init} from 'snabbdom';
```

Comment:
It uses `snabbdom` as a lightweight virtual dom lib. If you prefer another vdom lib, we can consider other options.
The vanilla JS version is, either, very very slow or unreadable (when optimized). Thus the idea of using a vdom lib.